scripts/deploy

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+build_chart() {
+  if [[ -z "${DOCKERHUB_USER}" ]]; then
+    echo "Please set env DOCKERHUB_USER to your dockerhub username"
+    exit 0
+  fi
+  #In the case of trying to build on non amd64 hosts, docker-buildx allows you to
+  #Cross compile for different architectures.
+  if [ "$ARCH" != "amd64" ]; then
+    echo "Building using docker-buildx"
+    docker buildx use default
+    make build
+    docker buildx use m1builder  
+    docker buildx build --platform linux/amd64 -f ../package/Dockerfile . -t $DOCKERHUB_USER/backup-restore-operator:$TAG --push
+
+  else
+    make build
+    docker build -f ../package/Dockerfile . -t $DOCKERHUB_USER/backup-restore-operator:$TAG --push
+  fi
+}
+
+create_cert() {
+  cat <<EOF > public.crt
+-----BEGIN CERTIFICATE-----
+MIICSTCCAe+gAwIBAgIQWgUVWCiZdyOGruNe6m4iWjAKBggqhkjOPQQDAjBMMRww
+GgYDVQQKExNDZXJ0Z2VuIERldmVsb3BtZW50MSwwKgYDVQQLDCNlbGl5YW1sZXZ5
+QEVsaXlhbXMtTUJQLmF0dGxvY2FsLm5ldDAeFw0yMjA1MTExNDAxMjBaFw0zMjA1
+MTEwMjAxMjBaMEwxHDAaBgNVBAoTE0NlcnRnZW4gRGV2ZWxvcG1lbnQxLDAqBgNV
+BAsMI2VsaXlhbWxldnlARWxpeWFtcy1NQlAuYXR0bG9jYWwubmV0MFkwEwYHKoZI
+zj0CAQYIKoZIzj0DAQcDQgAEFH8UPCl/vAHkMbTF3E8yhSdLNH2XueKUHns+O4FR
+hn096OJKnGZFb/HiW9iJWhj4CJ4LubSvsiZJZ7YuDlM9faOBsjCBrzAOBgNVHQ8B
+Af8EBAMCAqQwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDwYDVR0TAQH/BAUwAwEB/zAd
+BgNVHQ4EFgQUImjresql78fBpwSV7lp4fT4+NnwwWAYDVR0RBFEwT4IFbWluaW+C
+C21pbmlvLm1pbmlvgg9taW5pby5taW5pby5zdmOCHW1pbmlvLm1pbmlvLnN2Yy5j
+bHVzdGVyLmxvY2Fsgglsb2NhbGhvc3QwCgYIKoZIzj0EAwIDSAAwRQIgWT4CU5ib
+LNeXJmh2lnqEvaeKgqLHPFgMOQg+4TyO+uQCIQCI5WX1E84B+z6yX7WKIBYJIjto
+RjQi75QniF10pi2jKA==
+-----END CERTIFICATE-----
+EOF
+
+  cat <<EOF > private.key
+-----BEGIN PRIVATE KEY-----
+MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgRiROiBUTvzxfDbiu
+60w9WdlRKAI7Jy0c26uC2FoAZdahRANCAAQUfxQ8KX+8AeQxtMXcTzKFJ0s0fZe5
+4pQeez47gVGGfT3o4kqcZkVv8eJb2IlaGPgIngu5tK+yJklnti4OUz19
+-----END PRIVATE KEY-----
+EOF
+}
+
+deploy_minio() {
+  if [[ -z "${KUBECONFIG}" ]]; then
+   echo "Please set your Kubeconfig environment variable before running."
+   exit 0
+  fi
+
+  helm repo add minio https://charts.min.io/
+  helm repo update
+
+  echo "Creating Minio Namespace"
+  kubectl create ns minio
+
+  echo "Creating Minio tls-ssl secret"
+  kubectl -n minio create secret generic tls-ssl-minio --from-file=private.key --from-file=public.crt
+
+  echo "Installing Minio"
+  helm install --namespace minio --set rootUser=inspectorgadget,rootPassword=gogadgetgo --set tls.enabled=true,tls.certSecret=tls-ssl-minio --set replicas=1 --set resources.requests.memory=2Gi --set persistence.enabled=false --set drivesPerNode=0 --set pools=0 --set mode=standalone --set buckets[0].name=rancherbackups --set buckets[0].policy=none --set buckets[0].purge=true minio minio/minio
+
+  kubectl create secret generic miniocreds --from-literal=accessKey=inspectorgadget --from-literal=secretKey=gogadgetgo
+}
+
+k3s_deploy_minio() {
```

Comment:
We should make it as generic as possible but only test on k3s, so no dependencies on k3s.

Reply:
if you look theres two functions, one for k3s and one for testing. I can make them separate callable functions to generalize it?